.github/workflows/build.yaml

```diff
@@ -0,0 +1,11 @@
+name: Build
+on: [push, pull_request]
```

Comment:
This way there is only one build per pull request to master and one build in the master branch when merging anything (instead of two per pull request)
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```